saleor/graphql/translations/types.py

```diff
@@ -259,6 +264,10 @@ class ProductVariantTranslatableContent(ModelObjectType[product_models.ProductVa
     product_variant_id = graphene.ID(
         required=True, description="The ID of the product variant to translate."
     )
+    product_variant_sku = graphene.String(
```

Comment:
Since we already have name, the field could simply be called sku.
The product_variant_id field includes the product_variant prefix to distinguish it from the translation's id, but that doesn't seem necessary in this context.